modules/apps/asset/asset-list-api/src/main/java/com/liferay/asset/list/asset/entry/query/processor/AssetListAssetEntryQueryProcessor.java

```diff
@@ -22,6 +22,16 @@
  */
 public interface AssetListAssetEntryQueryProcessor {
 
+	public void processAssetEntryQuery(
```

Comment:
Should @lifera-echo review this pull?

Reply:
Good question! @dgarciasarai who was the reviewer of our first collab here?

Reply:
This was the main _Story_ for Content Recommendation: https://issues.liferay.com/browse/LPS-87232.
I agree with you, I think @liferay-echo should review this.

Reply:
I see It was by Edu in collaboration with @ealonso and @jorge.ferrer maybe It's good that some @liferay-echo dev takes a look ;)